src/Microsoft.SqlTools.ServiceLayer/sr.strings

```diff
@@ -243,8 +243,3 @@ BatchParser_VariableNotDefined = Variable {0} is not defined.
 # Workspace Service
 
 TestLocalizationConstant = EN_LOCALIZATION
```

Comment:
they were part of the original file committed here: https://github.com/Microsoft/sqltoolsservice/commit/41198e9357ebec8b1c2637d8719450b1031b30fb 
Looks like that was reverted somehow then reapplied by merging my branch.